experimenter/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_ready_for_review_serializer.py

```diff
@@ -1564,6 +1568,191 @@ def test_bucket_namespace_warning_for_experiments(self):
         self.assertTrue(serializer.is_valid())
         self.assertEqual(serializer.warnings, {})
 
+    def test_localized(self):
+        locale_en_us = LocaleFactory.create(code="en-US")
+        locale_en_ca = LocaleFactory.create(code="en-CA")
+        locale_fr = LocaleFactory.create(code="fr")
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.CREATED,
+            application=NimbusExperiment.Application.DESKTOP,
+            locales=[locale_en_us, locale_en_ca, locale_fr],
+            is_localized=True,
```

Comment:
So this test is testing the case where we have a localized experiment and provide some loclaes, but not explicitly passing in `localized_content`, which I guess is relying on the default value of the field in the factory.  In general a test should explicitly setup all of the priors it needs to pass, so I would expect to see `localized_content` explicitly declared here.